src/protojure/pedestal/interceptors/grpc.clj

```diff
@@ -68,11 +68,11 @@
   "<leave> interceptor for handling GRPC responses"
   [{:keys [server-streaming] :as rpc-metadata}
    {:keys [response] {:keys [req-ctx resp-ctx]} ::ctx :as context}]
-
   ;; special-case unary return types
   (when-not server-streaming
-    (let [output-ch (:in resp-ctx)]
-      (async/put! output-ch (:body response))
+    (let [output-ch (:in resp-ctx)
+          body (:body response)]
```

Comment:
minor nit, but the :in/:out stuff cant use destructuring easily because of the conflict between req-ctx and resp-ctx....but body can.  So, preferable to change L70 from `:keys [response]` to `{:keys [body]} :response`